### PR TITLE
Code and documentation changes to address review comments

### LINKED ISSUE #110: AMDirT commands always print a cache_data_api warning
```
(amdirt) james@bionb103:~$ AMDirT --help
2023-04-18 13:07:37.864 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.866 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.866 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.867 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.867 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.868 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.868 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
2023-04-18 13:07:37.869 WARNING streamlit.runtime.caching.cache_data_api: No runtime found, using MemoryCacheStorageManager
Usage: AMDirT [OPTIONS] COMMAND [ARGS]...

  AMDirT: Performs validity check of ancientMetagenomeDir datasets
  Authors: AMDirT development team and the SPAAM community
  Homepage & Documentation: https://github.com/SPAAM-community/AMDirT
  

Options:
  --version  Show the version and exit.
  --verbose  Verbose mode
  --help     Show this message and exit.

Commands:
  autofill  Autofills library and/or sample table(s) using ENA API and...
  convert   Converts filtered samples and libraries tables to eager,...
  merge     Merges new dataset with existing table 
  validate  Run validity check of ancientMetagenomeDir datasets...
  viewer    Launch interactive filtering tool
```

### PR DESCRIPTION
**Pull request by maxibor**
This also closes #110 by silencing the streamlit warning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,11 @@ pip install amdirt
...
 
+Installing AMDirT in a dedicated conda environment
+
 ```bash
-conda install -c bioconda amdirt
+conda create -n amdirt -c bioconda amdirt
+conda activate amdirt
 ```
 
 ### The latest development version, directly from GitHub
```
Comment (jfy133) [inline README.md:28]: Maybe mention for hardcore users who want to install everything manually (masochists) they can see `setup.py` for depedencies

File: AMDirT/core/__init__.py [hunk 2]
```diff
@@ -23,6 +23,17 @@
 logger.propagate = False
 
 
+def monkeypatch_get_storage_manager():
+    if st.runtime.exists():
+        return st.runtime.get_instance().cache_storage_manager
+    else:
+        # When running in "raw mode", we can't access the CacheStorageManager,
```
Comment (jfy133) [inline AMDirT/core/__init__.py:30]: Link to the URL where you took that info form?

File: README.md [hunk 3]
```diff
@@ -18,8 +18,11 @@ pip install amdirt
 
 ### 2. With conda
 
+Installing AMDirT in a dedicated conda environment
+
 ```bash
-conda install -c bioconda amdirt
+conda create -n amdirt -c bioconda amdirt
+conda activate amdirt
```
Comment (jfy133) [inline README.md:25]: Mention how to deactivate after

**Review by jfy133 — APPROVED** [review]
otherwise LGTM

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md:28]: Or just list the dependencies

File: README.md [hunk 4]
```diff
@@ -18,8 +18,11 @@ pip install amdirt
 
 ### 2. With conda
 
+Installing AMDirT in a dedicated conda environment
```
Comment (jfy133) [inline README.md:21]:
```suggestion
Installing AMDirT in a dedicated [conda](https://docs.conda.io/projects/miniconda/en/latest/index.html) environment
```

Please add a a corresponding link for pip installation

File: README.md (hunk 1 above)
Comment (maxibor) [reply README.md:28]: That's a bad idea, and creates a potential issue to keep the doc and the code in sync

File: AMDirT/core/__init__.py (hunk 2 above)
Comment (maxibor) [reply AMDirT/core/__init__.py:31]: Added the link to the issue, but this comment is litterally coming from the streamlit code itself.

File: README.md (hunk 4 above)
Comment (maxibor) [reply README.md:21]: Done

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md:30]: Yes agreed, after reading the comment of the reviewer again I forgot to delete this comment! Sorry!

**Pull request merged**